service-commander-def.yaml

```diff
@@ -0,0 +1,9 @@
+name: Manzan
```

Comment:
How does this file get consumed?

Reply:
Looks like I am missing a few steps actually to add this file to the global service commander definitions so that we can just use:

```sh
sc start manzan
```

Marking this as draft to fix this. I will also add a docs page about the installation and service commander steps.